app/src/main/java/com/codesoom/assignment/application/AuthenticationService.java

```diff
@@ -2,35 +2,72 @@
 
 import com.codesoom.assignment.domain.User;
 import com.codesoom.assignment.domain.UserRepository;
-import com.codesoom.assignment.errors.LoginFailException;
+import com.codesoom.assignment.dto.UserLoginData;
+import com.codesoom.assignment.errors.LoginInconsistencyException;
+import com.codesoom.assignment.errors.UnauthorizedException;
+import com.codesoom.assignment.errors.UserNotFoundException;
 import com.codesoom.assignment.utils.JwtUtil;
 import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.security.SignatureException;
 import org.springframework.stereotype.Service;
 
 @Service
 public class AuthenticationService {
-    private final UserRepository userRepository;
+
     private final JwtUtil jwtUtil;
+    private final UserRepository userRepository;
 
-    public AuthenticationService(UserRepository userRepository,
-                                 JwtUtil jwtUtil) {
-        this.userRepository = userRepository;
+    public AuthenticationService(JwtUtil jwtUtil, UserRepository userRepository) {
         this.jwtUtil = jwtUtil;
+        this.userRepository = userRepository;
     }
 
-    public String login(String email, String password) {
-        User user = userRepository.findByEmail(email)
-                .orElseThrow(() -> new LoginFailException(email));
+    /**
+     * 로그인을 검증합니다.
+     * @param loginData 로그인 데이터
+     * @return 로그인에 성공할 경우 토큰발급
+     */
+    public String login(UserLoginData loginData) {
```

Comment:
- main description은 리턴값 위주로 설명해 보세요.
- `@throws`는 빠뜨리지 않는 것이 좋습니다.

```suggestion
    /**
     * 로그인 요청 정보를 받아 로그인 처리를 한 다음, 인증 토큰을 리턴합니다.
     * @param loginData 로그인 요청 
     * @return 인증 토큰
     * @throws LoginInconsistencyException 로그인에 실패했을 경우
     */
    public String login(UserLoginData loginData) {
```

Reply:
반영 하였습니다 !